src/libsync/networkjobs.cpp

```diff
@@ -474,10 +474,7 @@ bool CheckServerJob::finished()
         }
 
         qDebug() << "status.php returns: " << status << " " << reply()->error() << " Reply: " << reply();
-        if( status.contains("installed")
-                && status.contains("version")
-                && status.contains("versionstring") ) {
-
+        if( status.contains("installed") ) {
```

Comment:
Yes. The presence of the other attributes is currently discussed at https://github.com/owncloud/core/issues/27669